src/swish/app.ms

```diff
@@ -396,7 +396,8 @@
               (cond
                [(whereis 'profiler) =>
                 (lambda (pid)
-                  (gen-server:call pid 'save 'infinity))]))
+                  (time
```

Comment:
You probably don't want to leave the time call in here?

Reply:
Oops! Thanks for catching this.